proposals/drafts/campaign_finance_filings.rst

```diff
@@ -0,0 +1,294 @@
+    they're involved. It will mostly be people but not always, and as lovely as
+    it would be to have some way to disambiguate the two I'm not optimistic that
+    will be possible anytime soon.
+
+Section
+    A container for a unit of meaning inside a Filing, that isn't inherent to
+    the basic process of filing a document with an agency.
+
+Rationale
+=========
+
+Political committees file statements with their regulators in order to disclose
+funders, expenditures, organizational ties and other financial details.
+Filings encapsulate some number of possible specific disclosures, and some
+filings can contain more than one type of disclosure, so this proposed option is
+to consider each actual filing as an object and then each specific disclosed
+piece of information as associated with that object. So a CommitteeNameChange
+would be associated with a Filing, along with a CommitteeAddressChange and a
+CommitteeOfficerAddition. A series of CommitteeContributions and
+CommitteeExpenses would be attached to one Filing. This way we don't need to
+model the specific disclosure forms in each state, and just directly extract
+the most meaningful parts of the filing.
+
+We want to make it easy for people to do manipulations on top of this data,
+while ensuring that this system itself does as little manipulation as possible
+to the underlying data from reports filed. For instance, many records include
+Person entities - these are deterministically extracted from just the available
+filing data, e.g. an address, employer, occupation and name are all extracted
+for each contribution. Deduplication can then happen with each of these
+entities, but they are themselves totally reproducible from just the filing
+information.
+
+This model entails that Filings have Sections - a Filing can, and often will,
+have multiple Sections. As much of the specific logic as possible should be
+pushed down into Sections, which offer flexibility by allowing us to compose
+them together to reproduce the data from state-specific reports. We can model
+a particular state filing form as a Filing with a specific set of expected
+Sections to be present. Some attributes that might make sense as properties of a
+Filing, such as Amendments, are here presented as Sections; the line between the
+two is muddy, but this proposal tries to keep Filings as minimal as possible.
+
+Finally, the different entity types presented here present either current or
+likely future conflicts with other entities in the Open Civic Data namespace, so
+this proposal doesn't make any attempt to resolve those conflicts since they
+seem ancillary for now.
+
+TODO
+----
+* Make filing activity actions use same pattern as OCD bill actions (@aepton)
+
+Questions to answer
+-------------------
+* How to model Committee Status and Committee Purpose inside of OCD Organization
+  models?
+
+Implementation
+==============
+
+Filing
```

Comment:
It would be nice if this description would be extensible. For US federal data, there are two main characteristics that determine what rules the filer has to abide by. For example, you need the committee type and committee designation to determine if a committee is a Super PAC. In addition to this, you can also see the org type to find organizations like labor unions, corporations or individuals.  
[Here is the FEC API's basic schema for committees](https://api.open.fec.gov/developers/#!/committee/get_committees), if that is helpful reference. We include all filers and not just those that are technically a committee.

Sometimes it makes sense to have election date if that is something you want to represent from a line on a form, like the [Form 3](http://www.fec.gov/pdf/forms/fecfrm3.pdf), but there are a lot of situations where this wouldn't apply. Especially in the primary season, on filing can apply to more than one election that will be on multiple days. Additionally, the election dates change, so if you add a day to each filing you will have to amend the election day on past filings or know that the same election can be represented as different days. We generally track the election type like primary, general, special or runoff election and the office that the election is for. You can then cross reference that election with a date from the date endpoint. 

I think that the election is generally more useful on the transaction level. That is where you can see if a donation is attributed to the primary or general etc., which is important for keeping track of donation limits per donor, since those are per-election. Again knowing the election type and office is better than having the date. 
For reference [here is the FEC API's basic schema for filings](https://api.open.fec.gov/developers/#!/filings/get_filings), though I would like to note, we would like to move toward a schema that separates the summary financial information from the filings information.

There are also some practical examples of how other smart people approch US federal data with the [fetch project](https://github.com/NYTimes/Fech)

I'm not nearly as knowledgeable about state or international campaigns, so feel free to ignore anything I say that isn't as widely applicable. (Also I am just commenting in my personal capacity.)